src/erlang_ls_methods.erl

```diff
@@ -136,7 +136,7 @@ exit(_Params, State) ->
   erlang_ls_utils:halt(ExitCode).
 
 %%==============================================================================
-%% textdocument_didopen
+%% textDocument/didopen
```

Comment:
These were probably erroneously replaced some time ago as part of a search/replace which passed review.